### DISCUSSION TITLE
百度网盘两个账户，只能加载一个，另一个出错！

### DISCUSSION THREAD

**Discussion by JohnchenYK** [AlistGo/alist#2936] [Invalid question]
如题
出错信息：
Failed init storage: Trigger security policy : Please try again later

**Comment by 65003118** [comment]
我和你有一样的问题，但是我的是一个账户挂在了不同IP的机器上

**Reply by zxdwhda** [reply]
我们一样

**Comment by yybshin** [comment]
需要怎么处理，我也遇到这样的问题了
